src/wt/main.js

```diff
@@ -1,5 +1,27 @@
+import { Worker } from 'worker_threads';
+import { cpus } from 'os';
+
+const inWorkerNthFibonacci = async (n) => {
+    return new Promise((resolve, reject) => {
+        const workerUrl = new URL('worker.js', import.meta.url);
+        const worker = new Worker(workerUrl, { workerData: n });
+        worker.on('message', resolve);
+        worker.on('error', reject);
+    });
+};
+
 const performCalculations = async () => {
-    // Write your code here
+    const INIT_COMPUTE = 10;
+    const promises = Array.from({ length: cpus().length })
```

Comment:
`os.cpus().length` should be replaced with [os.availableParallelism()](https://nodejs.org/docs/latest/api/os.html#osavailableparallelism)